xbmc/utils/FileUtils.cpp

```diff
@@ -33,6 +33,9 @@
 
 #include <vector>
 
+#define STR_HELPER(x) #x
+#define STR(x) STR_HELPER(x)
```

Comment:
I copied these from `xbmc/addons/kodi-dev-kit/include/kodi/versions.h`. I wasn't sure if I should try to share these somehow.